Tasks/UseDotNetV2/models.ts

```diff
@@ -87,11 +87,14 @@ export class Channel {
 }
 
 export class VersionParts {
```

Comment:
Please update its unit tests to test both cases specifically. In current tests explicitVersion = true is never being passed as the input did not exist earlier